Components/Sensors/Inc/Data.h

```diff
@@ -13,7 +13,6 @@
 
 typedef struct
 {
-    osMutexId   mutex_;
     int32_t     accelX_;
```

Comment:
remove postfix underscores?

Reply:
Let's change this later in a final cleanup